crates/subspace-farmer-components/src/farming.rs

```diff
@@ -143,7 +143,7 @@ impl EligibleSector {
                 sector_index: self.sector_index,
                 total_pieces: sector_metadata.total_pieces,
                 piece_offset: self.audit_piece_offset,
-                piece_record_hash: blake2b_256_254_hash_to_scalar(record.as_ref()),
+                piece_commitment_hash: blake2b_256_254_hash_to_scalar(commitment.as_ref()),
```

Comment:
I assume you will be updating farming later, but note that this will have to be `piece_commitment`